src/msg/async/rdma/RDMAStack.cc

```diff
@@ -125,16 +133,68 @@ void RDMADispatcher::polling()
 
   std::map<RDMAConnectedSocketImpl*, std::vector<ibv_wc> > polled;
   std::vector<ibv_wc> tx_cqe;
-  RDMAWorker* worker;
-  ldout(cct, 20) << __func__ << " going to poll rx cq:" << rx_cq << dendl;
+  ldout(cct, 20) << __func__ << " going to poll tx cq: " << tx_cq << " rx cq: " << rx_cq << dendl;
   RDMAConnectedSocketImpl *conn = nullptr;
   utime_t last_inactive = ceph_clock_now();
   bool rearmed = false;
-  int ret = 0;
+  int r = 0;
 
   while (true) {
-    int n = rx_cq->poll_cq(MAX_COMPLETIONS, wc);
-    if (!n) {
+    int tx_ret = tx_cq->poll_cq(MAX_COMPLETIONS, wc);
+    if (tx_ret > 0) {
+      ldout(cct, 20) << __func__ << " pool tx completion queue got " << tx_ret
+                     << " responses."<< dendl;
+      handle_tx_event(wc, tx_ret);
```

Comment:
you are correct, it's ok to handle Tx here, 
i see you removed some code from post_tx_buffer() that i was worried about